### PR TITLE
Use Picsum Images Instead of Lorem Pixel

### PR DESCRIPTION
**Pull request by thomasij813**
Updated `fakeDataGen.js` to use a selection of Picsum images instead of Lorem Pixel images. Lore Pixel provided the same image for each thumbnail. Furthermore, because Lorem Pixel urls are the same for all images, I couldn't use it as a unique key in my ImageSelection component.

### CODE REVIEW

File: fakeDataGen.js [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
 const outputArray = [];
 
-// Seed so we can reuse this file and get the same results
-faker.seed(100);
-
 // Create 100 records
 const numRecords = 100;
 for (let i = 1; i <= numRecords; i++) {
-  const numOfImages = faker.random.number({ min: 1, max: 10 });
+  const imageIds = [
```
Comment (JeehyaeDance) [inline fakeDataGen.js]: nit: I think it would be better to have these ids in one line, so you can see it at a glance?

File: fakeDataGen.js [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+    21,
+    20,
+    201,
+    2,
+    180,
+    160,
+    119
+  ];
+
+  const numOfImages = Math.floor(Math.random() * Math.floor(imageIds.length - 1)) + 1;
```
Comment (JeehyaeDance) [inline fakeDataGen.js]: do you need Math.floor for (imageIds.length - 1)? It think those will be integers even before the Math.floor.

File: fakeDataGen.js (hunk 1 above)
Comment (thomasij813) [reply fakeDataGen.js]: Yup, I agree it looks weird with them all spaced like this. I went ahead and put them all on one line, and I also changed const name from `imageIds` to `picsumImageIds` to make it more clear what this is an array of. I also included a comment explaining :)

File: fakeDataGen.js (hunk 2 above)
Comment (thomasij813) [reply fakeDataGen.js]: Nope, I don't, you're totally right. Good catch, thank you!

**Pull request merged**